static/js/apps/explore/app.tsx

```diff
@@ -229,7 +229,9 @@ export function App(props: { isDemo: boolean }): JSX.Element {
       }
     }
     const userMessage = {
-      msg: fulfillData["userMessage"] || "",
+      msg: !_.isEmpty(fulfillData["userMessages"])
+        ? fulfillData["userMessages"][0]
```

Comment:
maybe comment why we use first one?

Reply:
this change will be overwritten when https://github.com/datacommonsorg/website/pull/3910 goes in. This is just a temporary change so that there isn't any breakage in between the two PRs hitting autopush

Reply:
ah, makes sense.